Pod/Classes/LSSpan.m

```diff
@@ -38,207 +35,170 @@ - (NSDictionary*) toJSON:(NSUInteger)maxPayloadJSONLength {
 
 @end
 
-@implementation LSSpan {
-    LSTracer* m_tracer;
-    LSSpanContext* m_ctx;
-    LSSpanContext* m_parent;
-    NSString* m_operationName;
-    NSDate* m_startTime;
-    NSMutableArray<LSLog*>* m_logs;
-    NSMutableDictionary* m_tags;
-}
-
-- (instancetype) initWithTracer:(LSTracer*)client {
-    return [self initWithTracer:client
-                  operationName:@""
-                         parent:nil
-                           tags:nil
-                      startTime:nil];
-}
-
-- (instancetype) initWithTracer:(LSTracer*)tracer
-                  operationName:(NSString*)operationName
-                         parent:(nullable LSSpanContext*)parent
-                           tags:(nullable NSDictionary*)tags
-                      startTime:(nullable NSDate*)startTime {
+#pragma mark - LSSpan
+
+@interface LSSpan ()
```

Comment:
hm, can we just put `addTags` here? I don't think anything needs to call it outside of this .m file...